packages/example/pubspec.lock

```diff
@@ -5,126 +5,144 @@ packages:
     dependency: transitive
     description:
```

Comment:
@bensonarafat: discard the changes in this file and keep only the changes in the REAME.md

Reply:
alright